src/taipy/core/_manager/_manager.py

```diff
@@ -97,3 +97,10 @@ def _delete_entities_of_multiple_types(cls, _entity_ids: _EntityIds):
         _TaskManagerFactory._build_manager()._delete_many(_entity_ids.task_ids)
         _JobManagerFactory._build_manager()._delete_many(_entity_ids.job_ids)
         _DataManagerFactory._build_manager()._delete_many(_entity_ids.data_node_ids)
+
+    @classmethod
+    def _export(cls, id, **kwargs):
```

Comment:
instead of just the id, maybe we can expand this further and accept the entity object itself too

Reply:
and why do we use kwargs here instead of something like folder_path?

Reply:
1. We can always get the id with `entity.id` so what is the use of having the option to pass the entity?
2. kwargs is used to allow the export method to be fully generic at repository level. For example, if the scenario will be exported into a SQLite file if the SQLite repo is used. Similarly, the scenario will be exported into a collection if the MongoDB repo is used. Either way the paramater cannot be `folder_path`, and we may need more than one parameter.